modules/dxp/apps/ai-hub/ai-hub-rest-impl/src/main/java/com/liferay/ai/hub/rest/internal/resource/v1_0/TaskResourceImpl.java

```diff
@@ -57,17 +60,31 @@ public Task postTask(Task task) throws Exception {
 			throw new UnsupportedOperationException();
 		}
 
+		long groupId = WorkflowConstants.DEFAULT_GROUP_ID;
+
+		Scope taskScope = task.getScope();
```

Comment:
@MrJohn1911 rename to `scope`, I believe you named this way to avoid collisions in the DTO, if so you can create a method called `_toScope(group)`